test/fixtures/dialects/hive/select_interval.yml

```diff
@@ -121,10 +121,10 @@ file:
               literal: "'4'"
               date_part: SECOND
           - binary_operator: +
-          - keyword: INTERVAL
-          - literal: "'5'"
-          alias_expression:
-            identifier: NANO
```

Comment:
This was wrong. Now this is also recognized as date_part.

Reply:
Nice spot, yeah it's good to check things parse as expected to avoid later unexpected issues in rules/formatting 👍